api/aperture/policy/language/v1/classifier.proto

```diff
@@ -11,20 +11,32 @@ import "aperture/policy/language/v1/selector.proto";
 // See also [Classifier overview](/concepts/integrations/flow-control/flow-classifier.md).
 //
 // :::
-//
-// Example:
+// Example
 // ```yaml
-// selector:
+// flow_selector:
 //   service_selector:
-//     service: service1.default.svc.cluster.local
-//   flow_selector:
-//     control_point:
-//       traffic: ingress
+//      agent_group: demoapp
+//      service: service1-demo-app.demoapp.svc.cluster.local
+//   flow_matcher:
+//      control_point: ingress
+//      label_matcher:
+//         match_labels:
+//           user_tier: gold
+//         match_expressions:
+//           - key: user_type
+//             operator: In
+//             values:
+//               - insert
+//               - delete
+//           - label: user_agent
+//             regex: ".*Chrome.*"
+//
 // rules:
-//   user:
-//     extractor:
-//       from: request.http.headers.user
+//   extractor:
+//     from: request.http.headers.user-agent
+//   telemetry: false
 // ```
+
```

Comment:
remove the whitespace between the comment and the message - it's probably the reason why the documentation is deleted in markdown files